2021-09-13 16:59:15,505 - INFO     - server.py - Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . 
2021-09-13 16:59:35,897 - INFO     - server.py - Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . 
2021-09-13 17:01:40,523 - INFO     - server.py - Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . 
2021-09-13 17:02:02,761 - INFO     - server.py - Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . 
2021-09-13 17:06:49,774 - INFO     - server.py - Запущен сервер, порт для подключений: 7777, адрес с которого принимаются подключения: . 
2021-09-13 17:07:11,272 - INFO     - server.py - Запущен сервер, порт для подключений: 7778, адрес с которого принимаются подключения: . 
2021-09-13 17:07:33,819 - INFO     - server.py - Установлено соедение с ПК ('127.0.0.1', 52852) 
2021-09-13 17:07:33,850 - DEBUG    - server.py - Получено сообщение {'action': 'presence', 'time': 1631534853.8194008, 'user': {'account_name': 'Guest'}} 
2021-09-13 17:07:33,850 - DEBUG    - server.py - Разбор сообщения от клиента : {'action': 'presence', 'time': 1631534853.8194008, 'user': {'account_name': 'Guest'}} 
2021-09-13 17:07:33,866 - DEBUG    - server.py - Вызвана функция process_client_msg c параметрами ({'action': 'presence', 'time': 1631534853.8194008, 'user': {'account_name': 'Guest'}},), {}. Вызов из модуля __main__. Ver.1.Вызов из функции ).Ver.2.Вызов из функции main. 
2021-09-13 17:07:33,866 - INFO     - server.py - Cформирован ответ клиенту {'response': 200} 
2021-09-13 17:07:33,866 - DEBUG    - server.py - Соединение с клиентом ('127.0.0.1', 52852) закрывается. 
2021-09-13 17:13:22,649 - INFO     - server.py - Запущен сервер, порт для подключений: 7778, адрес с которого принимаются подключения: . 
2021-09-13 17:13:36,503 - INFO     - server.py - Установлено соедение с ПК ('127.0.0.1', 55402) 
2021-09-13 17:13:36,541 - DEBUG    - server.py - Получено сообщение {'action': 'presence', 'time': 1631535216.503833, 'user': {'account_name': 'Guest'}} 
2021-09-13 17:13:36,541 - DEBUG    - server.py - Разбор сообщения от клиента : {'action': 'presence', 'time': 1631535216.503833, 'user': {'account_name': 'Guest'}} 
2021-09-13 17:13:36,557 - DEBUG    - server.py - Вызвана функция process_client_msg c параметрами ({'action': 'presence', 'time': 1631535216.503833, 'user': {'account_name': 'Guest'}},), {}. Вызов из модуля __main__. Ver.1.Вызов из функции main().Ver.2.Вызов из функции main. 
2021-09-13 17:13:36,557 - INFO     - server.py - Cформирован ответ клиенту {'response': 200} 
2021-09-13 17:13:36,557 - DEBUG    - server.py - Соединение с клиентом ('127.0.0.1', 55402) закрывается. 
2021-09-13 17:41:17,404 - INFO     - server.py - Запущен сервер, порт для подключений: 7778, адрес с которого принимаются подключения: . 
2021-09-13 17:41:19,762 - INFO     - server.py - Установлено соедение с ПК ('127.0.0.1', 50897) 
2021-09-13 17:41:19,777 - DEBUG    - server.py - Получено сообщение {'action': 'presence', 'time': 1631536879.7623463, 'user': {'account_name': 'Guest'}} 
2021-09-13 17:41:19,777 - DEBUG    - server.py - Разбор сообщения от клиента : {'action': 'presence', 'time': 1631536879.7623463, 'user': {'account_name': 'Guest'}} 
2021-09-13 17:41:19,793 - DEBUG    - server.py - Вызвана функция process_client_msg c параметрами ({'action': 'presence', 'time': 1631536879.7623463, 'user': {'account_name': 'Guest'}},), {}. Вызов из модуля __main__. Ver.1.Вызов из функции main().Ver.2.Вызов из функции main. 
2021-09-13 17:41:19,793 - INFO     - server.py - Cформирован ответ клиенту {'response': 200} 
2021-09-13 17:41:19,793 - DEBUG    - server.py - Соединение с клиентом ('127.0.0.1', 50897) закрывается. 
2021-09-13 17:41:20,735 - INFO     - server.py - Установлено соедение с ПК ('127.0.0.1', 50898) 
2021-09-13 17:41:20,779 - DEBUG    - server.py - Получено сообщение {'action': 'presence', 'time': 1631536880.7355893, 'user': {'account_name': 'Guest'}} 
2021-09-13 17:41:20,779 - DEBUG    - server.py - Разбор сообщения от клиента : {'action': 'presence', 'time': 1631536880.7355893, 'user': {'account_name': 'Guest'}} 
2021-09-13 17:41:20,779 - DEBUG    - server.py - Вызвана функция process_client_msg c параметрами ({'action': 'presence', 'time': 1631536880.7355893, 'user': {'account_name': 'Guest'}},), {}. Вызов из модуля __main__. Ver.1.Вызов из функции main().Ver.2.Вызов из функции main. 
2021-09-13 17:41:20,779 - INFO     - server.py - Cформирован ответ клиенту {'response': 200} 
2021-09-13 17:41:20,779 - DEBUG    - server.py - Соединение с клиентом ('127.0.0.1', 50898) закрывается. 
2021-09-13 17:41:21,982 - INFO     - server.py - Установлено соедение с ПК ('127.0.0.1', 64278) 
2021-09-13 17:41:22,013 - DEBUG    - server.py - Получено сообщение {'action': 'presence', 'time': 1631536881.9825299, 'user': {'account_name': 'Guest'}} 
2021-09-13 17:41:22,013 - DEBUG    - server.py - Разбор сообщения от клиента : {'action': 'presence', 'time': 1631536881.9825299, 'user': {'account_name': 'Guest'}} 
2021-09-13 17:41:22,013 - DEBUG    - server.py - Вызвана функция process_client_msg c параметрами ({'action': 'presence', 'time': 1631536881.9825299, 'user': {'account_name': 'Guest'}},), {}. Вызов из модуля __main__. Ver.1.Вызов из функции main().Ver.2.Вызов из функции main. 
2021-09-13 17:41:22,013 - INFO     - server.py - Cформирован ответ клиенту {'response': 200} 
2021-09-13 17:41:22,013 - DEBUG    - server.py - Соединение с клиентом ('127.0.0.1', 64278) закрывается. 
